### PR TITLE
Fix client criteria links.

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-[![Client Capabilities](https://img.shields.io/badge/Kubernetes%20client-Silver-blue.svg?style=plastic&colorB=C0C0C0&colorA=306CE8)](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/csi-new-client-library-procedure.md#client-capabilities)
-[![Client Support Level](https://img.shields.io/badge/kubernetes%20client-alpha-green.svg?style=plastic&colorA=306CE8)](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/csi-new-client-library-procedure.md#client-support-level)
+[![Client Capabilities](https://img.shields.io/badge/Kubernetes%20client-Silver-blue.svg?style=plastic&colorB=C0C0C0&colorA=306CE8)](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/csi-new-client-library-procedure.md#client-capabilities)
+[![Client Support Level](https://img.shields.io/badge/kubernetes%20client-alpha-green.svg?style=plastic&colorA=306CE8)](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/csi-new-client-library-procedure.md#client-support-level)
```
Comment (lwander) [inline README.md]: I think we can call this beta now?

Comment (mbohlool) [reply README.md]: Second that.

**Comment by mbohlool** [comment]
should we close this in favor of https://github.com/kubernetes-client/java/pull/97?

**Comment by lwander** [comment]
sgtm

**Comment by mbohlool** [comment]
@brendandburns I created a new PR because I changed the links completely (used a bit.ly link so we can update it if these files moved again). I am going to close this PR.

**Pull request closed without merging**